quil-rs/src/instruction/gate.rs

```diff
@@ -128,13 +131,53 @@ impl fmt::Display for GateModifier {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, Eq, strum::Display, strum::EnumString)]
+#[strum(serialize_all = "UPPERCASE")]
+pub enum PauliWord {
+    I,
+    X,
+    Y,
+    Z,
+}
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct PauliTerm {
+    pub words: Vec<PauliWord>,
```

Comment:
I think some of the terminology is wrong here. A `PauliTerm` has a _single_ `PauliWord`, and a `PauliWord` is a string of characters drawn from the set `{I, X, Y, Z}`.

Reply:
Ah yeah, that makes sense. I've renamed the types, let me know what you think.